app/assets/stylesheets/common/base/activation.scss

```diff
@@ -9,6 +9,9 @@
     padding: 0;
     height: 100%;
   }
+  .above-main-container-outlet {
+    display: none;
+  }
```

Comment:
There's no easy way to remove the outlet in JS, right?

Reply:
Not really as it lives in `application.hbs`

https://github.com/discourse/discourse/blob/852ff18fb94f09c8fea2a2bd9f9277a5c27f4c51/app/assets/javascripts/discourse/app/templates/application.hbs#L53